src/forms/CheckboxGroup.scss

```diff
@@ -0,0 +1,15 @@
+.seeds-checkbox-group {
+  --inner-button-gap: var(--seeds-s3);
+  display: flex;
+  gap: var(--inner-button-gap);
+}
+
+.seeds-checkbox-hide {
+  opacity: 0;
+  position: absolute;
```

Comment:
You could reuse the `seeds-screen-reader-only` class for this; it also has clipping and stuff so there aren't weird click targets bunched up in the top-left of the screen.

Reply:
I tried this approach at first and could not get tab navigation to work with this class. Just tried again and saw similar results

Reply:
Hmm, interesting. Well perhaps you could just add the negative margin trick here, since we wouldn't want transparent click targets stacked up on the top-left of the page.

Reply:
Was this resolved? I wasn't seeing the transparent click targets.

Reply:
Got it to work!